src/infrastructure/validation/logger/LoggerFactory.java

```diff
@@ -1,11 +1,10 @@
-/**
- * 
- */
 package infrastructure.validation.logger;
 
 /**
+ * LoggerFactory class provides the method getLoggerInstance()
+ * This method will return a singleton reference that can be used for logging
```

Comment:
What object's singleton instance is being returned?

Reply:
actually the loggerManager. but this info isn't really required to be disclosed. For a client, better description is 
> This method will return a singleton reference to the object that implements ILogger which can then be used for logging

I've changed accordingly

Reply:
finally, it reads like this:
> LoggerFactory class provides the method getLoggerInstance(). The approach followed is the factory pattern. This method will return a singleton reference to the object, that implements ILogger interface and can so, be used for logging

Reply:
Well the client shouldn't be reading and trying to understand source code but a developer will. Your new description in
that case is fine.